about.html

```diff
@@ -0,0 +1,158 @@
+          expedita illum consectetur tempore veritatis ad? Asperiores quia magni
+          in obcaecati suscipit eos neque quisquam corrupti necessitatibus.
+          Labore quae molestiae totam. Incidunt, autem minima facere provident
+          est earum? Ab corporis architecto perspiciatis perferendis, eveniet
+          dolorem recusandae dolores quam libero quidem delectus iusto nesciunt
+          fuga aperiam aspernatur et maxime soluta, nostrum reprehenderit totam,
+          quibusdam asperiores consequuntur optio. Iste unde enim voluptatibus
+          doloribus rerum optio.
+        </p>
+      </div>
+      <div id="conference-contact">
+        <h5>
+          Please contact us per Email for further details about Fan Expo San
+          Francisco 2023!
+        </h5>
+        <p>fanexpo2023@fanexposf.com</p>
+      </div>
+    </section>
+    <section class="conference-section">
+      <div>
+        <div id="conf-sect-name">
+          <h3>Anime & Comicon <br />Fan Expo 2023 Logo</h3>
+        </div>
+        <hr />
+        <div id="conf-sect-detail">
+          <p>
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Mollitia,
+            est.
+          </p>
+        </div>
+        <div id="conf-sect-logo">
+          <img src="./Images/Logo.png" alt="Logo" />
+        </div>
+      </div>
+      <hr id="hr3" />
+    </section>
+    <section class="past-expo">
+      <div id="past-expo-info">
+        <h3>See the past Anime Fan Expos</h3>
+        <hr />
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Vero, quam.
+        </p>
+      </div>
+      <div class="expo-details">
+        <div id="expo-2020">
+          <div class="image-overlay">
+            <div id="expo-detail">
+              <h3>2020</h3>
+              <p>FanExpo Gathering 2020</p>
+            </div>
+          </div>
+        </div>
+        <div id="expo-2021">
+          <div class="image-overlay">
+            <div id="expo-detail">
+              <h3>2021</h3>
+              <p>FanExpo Visitors 2021</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="partners-mobile">
+      <div id="partners-mobile">
+        <h3>Partners</h3>
+        <hr />
+      </div>
+      <div class="logos">
+        <img src="./Images/Marvel-logo.png" alt="marvel-logo" />
+        <img src="./Images/DC-logo.svg" alt="dc-logo" />
+        <img src="./Images/Toei_Animation_logo.png" alt="toei-logo" />
+        <img src="./Images/Studio_Pierrot-logo.webp" alt="pierrot-logo" />
+        <img src="./Images/studio-ghibli-logo.jpeg" alt="ghibli-logo" />
+      </div>
+    </section>
+    <footer class="footer-mobile">
+      <div id="footer-logo">
+        <img src="./Images/Logo.png" alt="logo" id="footer-img" />
+      </div>
+      <div id="footer-info">
+        <p>Lorem ipsum dolor sit amet consectetur</p>
+      </div>
+    </footer>
```

Comment:
- [ ] On the guiding design, the background color is black, but your design carries something different. Kindly look into that and fix.
- [ ] You should also keep in mind the spacing of your footer compare to the guiding template.

### Your Design (Desktop)
![your-about-footer-desktop-bg](https://user-images.githubusercontent.com/26597432/212051885-2614ab25-b8fd-4a17-864d-bdfc75862bcc.png)

### Guiding Template (Desktop)
![about-footer-desktop-bg](https://user-images.githubusercontent.com/26597432/212051958-044e5ae6-de15-4f85-bf75-0d11b6277299.png)